util/test/perf/perfgraph.js

```diff
@@ -545,6 +545,12 @@ function computeGitHubLinks(text) {
     return "<a target='_blank' href='" + url + "'>" + m + "</a>";
   });
 
+  var private_re = /\(Cray\/chapel-private#([0-9]+)\)/gi;
+  text = text.replace(private_re, function(m, num) {
+    var url = "https://github.com/Cray/chapel-private/issues/" + num;
+    return "<a target='_blank' href='" + url + "'>" + m + "</a>";
+  });
+
```

Comment:
Good candidate for some code reuse, but this is totally fine. I'll push harder when we add a third case like this :)